DataSource/DataSourceCellDescriptor.swift

```diff
@@ -106,6 +106,9 @@ extension TableViewDataSource {
 		}
 		tableView.dataSource = self
 		self.tableView = tableView
+		if #available(iOS 11.0, *) {
+			tableView.performBatchUpdates(nil)
```

Comment:
What is this for?

Reply:
@vadim-fueled I'm aligning the approach with the solution applied for CollectionViewDataSource `configure` method recently. Without this change such simple piece of code will crash with a `NSInternalInconsistencyException`:
```
inal class TestViewController: UIViewController {
	@IBOutlet private var tableView: UITableView!
	private let dataSource = TableViewDataSource()

	override func viewDidLoad() {
		super.viewDidLoad()

		self.dataSource.configure(
			self.tableView,
			using: [
				CellDescriptor("DefaultCell", String.self),
			])

		self.dataSource.dataSource.innerDataSource.value = StaticDataSource(items: ["One", "Two", "Three"])
	}
}
```